### PR TITLE
ci: add copy and publish steps

### LINKED ISSUE #79: Add latests build to release page
Pls add latests build to [release](https://github.com/appium/io.appium.settings/releases) page

### PR DESCRIPTION
**Pull request by fleytman**
I create PR for test build on CI
to fix #79

### CODE REVIEW

File: .azure-templates/gradle_job.yml [hunk 1]
```diff
@@ -23,3 +23,12 @@ jobs:
       jdkVersionOption: "1.${{ parameters.jdk }}"
       jdkArchitectureOption: 'x64'
       tasks: "${{ parameters.task }}"
+  - task: CopyFiles@2
+  displayName: 'Copy Files to: $(build.artifactStagingDirectory)'
+    inputs:
+      Contents: '**/*.apk'
```
Comment (KazuCocoa) [inline .azure-templates/gradle_job.yml:29]: The path to the debug build for release is `apks/settings_apk-debug.apk`. Perhaps could `apks/settings_apk-debug.apk` be enough?

`build.artifactStagingDirectory` should be `Build.ArtifactStagingDirectory`?

Comment (fleytman) [reply .azure-templates/gradle_job.yml:29]: Go test with '**/*.apk'. If that doesn't work, let's try the absolute path.

**Comment by fleytman** [comment]
I work: https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=20263&view=artifacts&pathAsName=false&type=publishedArtifacts
As I understand it, artifacts should be automatically pulled into the release by a trigger. I'll have to check it after merge.

**Comment by KazuCocoa** [comment]
thank you.
I'll merge this and add a dummy commit to kick the release task how it will be tonight.

btw, the release task is https://github.com/appium/io.appium.settings/actions/workflows/publish.js.yml

**Review by KazuCocoa — APPROVED** [review]

**Comment by KazuCocoa** [comment]
ah, i thought this was GHA, but azure one.
https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=20271&view=results is the result

**Pull request merged**